src/modules/PublicLab.TitleModule.Related.js

```diff
@@ -35,28 +38,73 @@ module.exports = function relatedNodes(module) {
 
     module.el.find('.ple-module-content').append('<div style="display:none;" class="ple-title-related"></div>');
     relatedEl = module.el.find('.ple-title-related');
-    relatedEl.append('<p class="ple-help">Does your work relate to one of these? Click to alert those contributors.</p><hr style="margin: 4px 0;" />');
+    relatedEl.append('<p class="ple-help">Does your work relate to one of these? Click to tag the related work.</p><hr style="margin: 4px 0;" />');
+    module.el.find('.ple-module-content').append('<div style="display:none;" class="ple-title-added"></div>');
+    addedRelatedEl = module.el.find('.ple-title-added');
 
   }
 
+  function showAdded(addedResult, relatedResults) {
+    addedRelatedEl.append('<div class="addedresult addedresult-' + addedResult.id + '" style="margin: 3px;"><a class="btn btn-xs btn-default remove-tag"><i class="fa fa-times-circle"></i> Remove</a> <a class="addedtitle"></a> by <a class="addedauthor"></a></div>');
+    addedRelatedEl.find('.addedresult-' + addedResult.id + ' .addedtitle').html(addedResult.title);
+    addedRelatedEl.find('.addedresult-' + addedResult.id + ' .addedtitle').attr('href', addedResult.url);
+    addedRelatedEl.find('.addedresult-' + addedResult.id + ' .addedauthor').html('@' + addedResult.author);
+    addedRelatedEl.find('.addedresult-' + addedResult.id + ' .addedauthor').attr('href', '/profile/' + addedResult.author);
+
+    $('.addedresult-' + addedResult.id + ' .remove-tag').click(function() {
```

Comment:
This block can be refactored in a standalone function and documented since it's quite large.